src/index.html

```diff
@@ -3,12 +3,49 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body> 
+    <header class="header"> 
+      <div class="logo"> 
+        <a href="/" class="logo-link"> 
```

Comment:
Use anchor links instead. Fix it everywhere 

https://github.com/mate-academy/layout_moyo-header/assets/99403198/b68ea77f-6ea7-4252-a840-057b072f295a